lib/periodic_jobs.rb

```diff
@@ -149,7 +149,9 @@
   # Daily/weekly report of unsuccessful benefits intake submissions
   mgr.register('0 2 1 * *', 'VBADocuments::ReportMonthlySubmissions')
   # Monthly report of benefits intake submissions
-  mgr.register('0 2,9,16 * * 1-5', 'VBADocuments::SlackNotifier')
+  mgr.register('0 8,12,17 * * 1-5', 'VBADocuments::SlackNotifier')
+  # Notifies slack channel if BI uploads start to expire with no S3 file uploads provided
+  mgr.register('15 * * * *', 'VBADocuments::SlackStatusNotifier')
```

Comment:
A couple of things here:

1. It looks like the job descriptions on L153 and L155 are out of order. Descriptions should be under their job definitions, not above, to keep in line with the rest of the file.
2. The description for `SlackStatusNotifier`, `# Notifies slack channel if BI uploads start to expire with no S3 file uploads provided`, appears out of date since it does more than one thing now (alerting on expired and "stuck in uploaded" submissions).
3. In a DM, you shared your intent to pull the rest of the status checks out of `SlackNotifier` and into `SlackStatusNotifier`. Right now, based on the current names and their descriptions, I can't tell why both jobs exist or how they're different. Can you update the job names or the descriptions to more clearly communicate the intent of these jobs?

Reply:
> 1. It looks like the job descriptions on L153 and L155 are out of order. Descriptions should be under their job definitions, not above, to keep in line with the rest of the file.
Fixed

> 2. The description for SlackStatusNotifier, # Notifies slack channel if BI uploads start to expire with no S3 file uploads provided, appears out of date 
Fixed

>3. In a DM, you shared your intent to pull the rest of the status checks out of SlackNotifier and into SlackStatusNotifier. Right now, based on the current names and their descriptions, I can't tell why both jobs exist or how they're different. Can you update the job names or the descriptions to more clearly communicate the intent of these jobs?

I changed the name and description of SlackNotifier to SlackInflightNotifier to indicate that it monitors submissions that have been sent to our downstream for processing\intake.

Created a follow up ticket to consider the future of SlackInflightNotifier:
https://jira.devops.va.gov/browse/API-36678